tests/playwright/tests/columnQrCode.spec.ts

```diff
@@ -0,0 +1,103 @@
+        },
+      ];
+
+      // close 'Team & Auth' tab
+      await dashboard.closeTab({ title: 'Team & Auth' });
+
+      await dashboard.treeView.openTable({ title: 'City' });
+
+      await dashboard.grid.column.create({
+        title: 'QrCode1',
+        type: 'QrCode',
+        qrCodeValueColumnTitle: 'City',
+      });
+
+      await qrCodeColumnVerify('QrCode1', expectedQrCodeCellValues);
+
+      // Clicking on qr code in first row and expect it shows a
+      // popup with an enlarged version of the qr code
+      await dashboard.grid.cell.get({ columnHeader: 'QrCode1', index: 0 }).click();
+      const qrGridOverlay = dashboard.grid.qrCodeOverlay;
+      await qrGridOverlay.verifyQrValueLabel(expectedQrCodeCellValues[0].referencedValue);
+      await qrGridOverlay.clickCloseButton();
+
+      await dashboard.closeTab({ title: 'City' });
+    });
+
+    test('deletion when the reference value column is deleted', async () => {
+      // close 'Team & Auth' tab
```

Comment:
Add test case for the case of editing the column:
- Column title update
- Column change(and verify the qr cell)

Reply:
same as above